config/locales/time/es.yml

```
@@ -2,6 +2,7 @@
 es:
   date:
     day_names:
+      -
```

ditto re: comment on empty line?
```suggestion
      - # empty item to have correct weekday offset
```